Makefile

```diff
@@ -43,16 +43,16 @@ lint:
 
 #Test
 TEST_DB = vulcanize_testing
-TEST_CONNECT_STRING = postgresql://localhost:5432/$(TEST_DB)?sslmode=disable
+TEST_CONNECT_STRING = postgresql://postgres@localhost:5432/$(TEST_DB)?sslmode=disable
 
 .PHONY: test
 test: | $(GINKGO) $(LINT)
 	go vet ./...
 	go fmt ./...
 	dropdb --if-exists $(TEST_DB)
 	createdb $(TEST_DB)
-	$(GOOSE) -dir db/migrations "$(TEST_CONNECT_STRING)" up
-	$(GOOSE) -dir db/migrations "$(TEST_CONNECT_STRING)" reset
+	$(GOOSE) -dir db/migrations postgres "$(TEST_CONNECT_STRING)" up
+	$(GOOSE) -dir db/migrations postgres "$(TEST_CONNECT_STRING)" reset
```

Comment:
This requires that the use has a `postgres` database user, right? I think in our README we offer a way to use a different user than `postgres`. Do you think it's worth allowing that here too? I could go either way.

Reply:
This line is just specifying `postgres` as the driver for goose, though the `TEST_CONNECT_STRING` has the user hard coded as `postgres`. Happy to enable configuring the user in the tests' connection string 👍

Reply:
0e7475ba3a80ddfe7d14cd0f918a70ab56e21346